### ISSUE TITLE
Cannot change the default payment to customer 

### ISSUE THREAD

**Issue by ReasonAmu** [stripe/stripe-ios#1785]
## Summary
i can't change default payment method by update customer method. I used to STPCustomerContext but it is STPAddress param.
I found STPAPIClient have update customer method, but i can't call update customer method in my project, it is internal [structure.]
How to update default payment method of customer? 
## iOS version
iOS 14.4, Xcode 12.4
## Installation method
Cocoapods
## SDK version
21.2.1
## Other information
i want update default payment method in customer
https://stripe.com/docs/api/customers/update

<img width="642" alt="Screen Shot 2021-03-30 at 09 37 39" src="https://user-images.githubusercontent.com/13688116/113238567-3b904f00-92d3-11eb-8fbe-07ce37f38b43.png">
<img width="702" alt="Screen Shot 2021-03-30 at 09 28 36" src="https://user-images.githubusercontent.com/13688116/113238591-477c1100-92d3-11eb-9d6a-3b5ebe19978f.png">

**Comment by mvarie** [comment]
@ReasonAmu , not sure that this is what you're looking for, but the `STPCustomerContext` class has a `attachPaymentMethodToCustomer`  public function:

https://github.com/stripe/stripe-ios/blob/b33030f6c2110f7370b04da7ae46589009c3fd02/Stripe/STPCustomerContext.swift#L227-L261

**Comment by csabol-stripe** [comment]
Hi @ReasonAmu , Customers currently only have default payment methods for Invoicing, which is not currently supported in the mobile SDKs. Is that what you're trying to update?

**Comment by ReasonAmu** [comment]
> Hi @ReasonAmu , Customers currently only have default payment methods for Invoicing, which is not currently supported in the mobile SDKs. Is that what you're trying to update?

Yep, in app design of me, user have switch default payment method in list payment method of them. It is custom UI follow style app, so i don't use default stripe UI.

I saw stripe sdk android have set default payment method pass params "source" update customer, but stripe sdk ios can't apply

Can you suggest solution for me?
<img width="690" alt="Screen Shot 2021-04-04 at 02 59 15" src="https://user-images.githubusercontent.com/13688116/113490097-47a62780-94f2-11eb-979a-31859c138a12.png">
